src/qml/qmlconfigproxy.h

```diff
@@ -18,8 +18,11 @@ class QmlConfigProxy : public QObject {
             UserSettingsPointer pConfig,
             QObject* parent = nullptr);
 
+    // We use method here instead of properties as there is no way to achieve property binding
+    // with UserSettings, since there is no synchronisation upon mutations.
```

Comment:
Added that comment as I was tempted to refactor this to `Q_PROPERTY` at first glance, but then I remember the static aspect of user settings at the moment, so thought I will put the thought here to save someone else the same temptations :)